### ISSUE TITLE
badges show on docs.ropensci.org for some pkgs

### ISSUE THREAD

**Issue by dpprdan** [ropensci-org/rotemplate#100]
For some packages, e.g. opencage, the badges in the [README.md](https://github.com/ropensci/opencage/blob/ad7ff7d102b47ce266146c312a2665a57922c3ae/README.md) don't show on https://docs.ropensci.org/opencage/ (or only the one for peer-review in the sidebar).

Other examples are:

- [gert](https://docs.ropensci.org/gert/)
- [awardFindR](https://github.com/ropensci/awardFindR)
- [cffr](https://docs.ropensci.org/cffr/)
- [gitignore](https://docs.ropensci.org/gitignore/)

I (vaguely) suspect the `<!-- badges: start -->` comments to be the culprit as that seems to be the common denominator of the repos above.

**Comment by jeroen** [comment]
I think this is intended? We only show the software review badge now in the docs page: https://github.com/ropensci-org/rotemplate/issues/95

**Comment by maelle** [comment]
yes, it's a choice in pkgdown to not show badges on production websites, and we re-add the software review badge.

**Comment by dpprdan** [comment]
That's also great, but then it is inconsistent, because there are badges (in the main page, not the sidebar) on e.g.:

- [bikedata](https://docs.ropensci.org/bikedata/ "Download and Aggregate Data from Public Hire Bicycle Systems • bikedata")
- [codemetar](https://docs.ropensci.org/codemetar/ "Generate CodeMeta Metadata for R Packages • codemetar")
- [bomrang](https://docs.ropensci.org/bomrang/ "Australian Government Bureau of Meteorology (BOM) Data Client • bomrang")
- [crul](https://docs.ropensci.org/crul/ "HTTP Client • crul")
- [pdftools](https://docs.ropensci.org/pdftools/ "Text Extraction, Rendering and Converting of PDF Documents • pdftools")

pkgdown only filters out the `<!-- badges -->` block then, I assume. (I guess, we could add a note to use the comments to [devguide here](https://devguide.ropensci.org/building.html?q=badges#readme)?)

All in all not a big issue, though, so we can close this, IMO.

**Comment by maelle** [comment]
No, you're right, its annoying. One of these days I should probably do a round and make PRs to fix a bunch of READMEs. :sweat_smile:

**Comment by dpprdan** [comment]
> pkgdown only filters out the `<!-- badges -->` block then, I assume.

Uh, maybe not. Counterexample: [rotl](https://docs.ropensci.org/rotl/ "Interface to the Open Tree of Life API • rotl") has badges, and a [`<!-- badges -->` block](https://github.com/ropensci/rotl/blob/0cacf9e81431f9bf584843a5bc1e9d20b2999c69/README.md?plain=1#L2-L15).

**Comment by maelle** [comment]
https://pkgdown.r-lib.org/reference/build_home.html#dev-badges

**Comment by dpprdan** [comment]
> [pkgdown.r-lib.org/reference/build_home.html#dev-badges](https://pkgdown.r-lib.org/reference/build_home.html#dev-badges)

I _think_ rotl passes those requirements (I read them as "or" reqs). Maybe they show up because the badges block comes before the title?

**Comment by maelle** [comment]
it might be https://github.com/r-lib/pkgdown/issues/2060

**Comment by maelle** [comment]
For info https://github.com/r-lib/pkgdown/pull/2220 :slightly_smiling_face:
